staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3976,6 +4002,13 @@ type EphemeralContainerCommon struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container to manage the restart behavior of each
+	// container within a pod.
+	// This may only be set for init containers. You cannot set this field on
```

Comment:
If it can't be set, why are we adding it?

Reply:
https://github.com/kubernetes/kubernetes/blob/8d4479a58f0b1864bdc0ec386e045239b89d15d2/staging/src/k8s.io/api/core/v1/types.go#L4055-L4057

^because of this.

Reply:
We probably need something that hides fields from the api docs.  This is fine.